examples/react_webpack_hotreload/WORKSPACE

```diff
@@ -0,0 +1,22 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "10fffa29f687aa4d8eb6dfe8731ab5beb63811ab00981fc84a93899641fd4af1",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/2.0.3/rules_nodejs-2.0.3.tar.gz"],
+)
+
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
+
+
+node_repositories(
+    node_version = "12.18.3"
```

Comment:
is this load-bearing? if the default node version works, I'd rather make the example minimal so that every line is instructive

Reply:
nope, left over from me needing multiple node versions